### DISCUSSION TITLE
Is it safe to copy the entire electrs db on a new machine to avoid re-index?

### DISCUSSION THREAD

**Discussion by sanket1729** [romanz/electrs#1010] [Q&A] [answered]
Are there any pitfalls that I should be aware of? I do not want to end up in a state where something in not indexed.

**Comment by Kixunil** [comment, marked as answer]
Your question basically boils down to "is it safe to copy rocksdb"? Quick search didn't find anything other than not recommending copying an open database (so just stop electrs before copying). If both machines are of the same architecture I don't see any reason why it'd be unsafe.

**Reply by sanket1729** [reply]
Tried it and it worked
